WordPress/Classes/Utility/CoreDataHelper.swift

```diff
@@ -434,15 +434,41 @@ extension CoreDataStack {
     }
 }
 
+/// `CoreDataObjectIdentifier` is a strongly typed `NSManagedObject`.
+///
+/// By using this strongly typed `NSManagedObject`, we can declare APIs that're bound to specific model types, thus
+/// preventing an incorrect object id being used as argument.
+///
+/// Take the following function as an example,
+/// ```
+/// func getPost(fromBlogID blogID: NSManagedObjectID)
+/// ```
```

Comment:
For the record, I tried annotating this code block with `swift` and adding a new line to see if it rendered better in quick help, but it didn't change.

<img width="485" alt="Screenshot 2023-07-25 at 12 02 00 pm" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/1218433/d670466e-f14d-47fb-9a14-2deadc148c90">